lib/hardware_services/quads_default_driver.py

```diff
@@ -0,0 +1,2 @@
+# this class will inherit from hardware_service.py and overwrite all of its methods
```

Comment:
@vsemp while they may not directly interact with the hardware in the current quads implementation, I think they are still required to manage hardware isolation. Therefore, they should still be included in the hardware_service class, right? What do you think?